_posts/2012-04-18-app.markdown

```diff
@@ -340,7 +340,7 @@ to
 <%= form.file_field :picture %>
 {% endhighlight %}
 
-In your file it might actually say `<%= f.text_field :picture %>`, just roll with it.
+In your file it might actually say `<%= form.text_field :picture %>`, just roll with it.
```

Comment:
I think, it was for backward compatibility. Previously, it used to come `f` instead of `form`.
I believe this line shouldn't be there now.

Reply:
So you would rather srecommend to remove it at all?